.gitignore

```
@@ -1,7 +1,17 @@
 qemu/
 virtualbox-iso/
 vmware-iso/
-
+packer_plugins/
+ansible/collections
+ansible/roles/galaxyproject*
+ansible/roles/geerlingguy*
+ansible/roles/influxdata*
+ansible/roles/usegalaxy-eu.a*
+ansible/roles/usegalaxy_eu.*
+ansible/roles/usegalaxy-eu.c*
+ansible/roles/usegalaxy-eu.d*
+ansible/roles/usegalaxy-eu.f*
+ansible/roles/usegalaxy-eu.t*
```

I would rather either do this,

```suggestion
packer_plugins/
ansible/collections
```

or this.

```suggestion
packer_plugins/
ansible/collections
ansible/roles/
```

In the first setup you are responsible for keeping your environment clean and/or taking care of properly preparing commits.

In the second setup you would ignore .gitignore when adding the roles.

```shell
[jose@BILP51 vgcn]$ git add packer_cache/abcd.txt 
Die folgenden Pfade werden durch eine Ihrer ".gitignore" Dateien ignoriert:
packer_cache
Hinweis: Nutzen Sie die Option -f, wenn sie wirklich hinzugefügt werden sollen.
Hinweis: Um diese Meldung abzuschalten, führen Sie folgenden Befehl aus:
Hinweis: "git config advice.addIgnoredFile false"
```